frontends/web/src/routes/settings/components/device-settings/root-fingerprint-setting.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { SettingsItem } from '../settingsItem/settingsItem';
+
+type TProps = {
+    rootFingerprint: string;
+}
+
+const RootFingerprintSetting = ({ rootFingerprint }: TProps) => {
+  const { t } = useTranslation();
+  return (
+    <SettingsItem
+      settingName={'Root fingerprint'}
+      secondaryText={t('deviceSettings.deviceInformation.rootFingerprint.description')}
+      displayedValue={rootFingerprint}
+    />
```

Comment:
Since this is quiet a long text, please add:

```diff
diff --git a/frontends/web/src/routes/settings/components/settingsItem/settingsItem.module.css b/frontends/web/src/routes/settings/components/settingsItem/settingsItem.module.css
index 89d455103..889ad050b 100644
--- a/frontends/web/src/routes/settings/components/settingsItem/settingsItem.module.css
+++ b/frontends/web/src/routes/settings/components/settingsItem/settingsItem.module.css
@@ -32,6 +32,7 @@
 .rightContentContainer {
     display: flex;
     align-items: center;
+    flex-shrink: 0;
 }
 
 .displayedValue {
```

so it doesn't break the value:

![Screen Shot 2023-07-04 at 15 35 11](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/b379d636-00a4-4480-acc5-2a7850238f96)

Reply:
Thanks, done.